### ISSUE TITLE
Mac M1 advisory db fetch failure

### ISSUE THREAD

**Issue by repi** [EmbarkStudios/cargo-deny#330]
Running simply `cargo deny check` on any repo after building it on my MBP13 M1 for `aarch64-apple-darwin` and Rust 1.50 fails for some reason for me. Not sure if aarch64/M1 specific or just general issue on this Mac somehow.

```
$ cargo deny check
2021-02-12 19:11:49 [ERROR] failed to fetch advisory database: git operation failed: failed to lock file '/Users/repi/.cargo/advisory-dbs/github.com-dfdf83aac62a7e5e/.git/refs/remotes/origin/master.lock' for writing: ; class=Os (2)
```

Any ideas?

**Comment by maxburke** [comment]
For what it's worth I'm seeing a similar issue on Ubuntu 20.04

```
$ cargo deny check
2021-02-14 19:43:28 [ERROR] failed to fetch advisory database: git operation failed: an unknown git error occurred
```

**Comment by Jake-Shadle** [comment]
I can't merge https://github.com/EmbarkStudios/cargo-deny/tree/update yet due to https://github.com/RustSec/rustsec-crate/issues/294, but it will be fine on either mac or ubuntu, it _might_ address the issue since both git2 and rustsec are being updated, but I'm just guessing since I have never seen this particular issue.

**Comment by repi** [comment]
@Jake-Shadle was it PR #333 you were referring to? which has been merged now recently.

did test installing the latest cargo-deny 0.8.8, but still have the same problem.

**Comment by Jake-Shadle** [comment]
Ok, I guess I will have to test this on our Mac ci machine.

**Comment by Jake-Shadle** [comment]
Using the latest code built with 1.51.0 I was unable to repro this on an M1...maybe addressed by latest git2 upgrades?

**Comment by bnjbvr** [comment]
```
40345eedf3fd6d4a418556f04f559ed94121de8d is the first new commit
commit 40345eedf3fd6d4a418556f04f559ed94121de8d
Author: Jake Shadle <jake.shadle@embark-studios.com>
Date:   Mon Mar 8 09:45:47 2021 +0100

    Fix rustsec/advisory-db (#336)

 CHANGELOG.md | 3 +++
 Cargo.lock   | 4 ++--
 2 files changed, 5 insertions(+), 2 deletions(-)
```

Fixed by 40345eedf3fd6d4a418556f04f559ed94121de8d.
